lib/JsonpChunkTemplatePlugin.js

```diff
@@ -12,20 +12,20 @@ JsonpChunkTemplatePlugin.prototype.apply = function(chunkTemplate) {
 	chunkTemplate.plugin("render", function(modules, chunk) {
 		var jsonpFunction = this.outputOptions.jsonpFunction;
 		var source = new ConcatSource();
-		source.add(jsonpFunction + "(" + JSON.stringify(chunk.ids) + ",");
+		source.add("(window[" + JSON.stringify(jsonpFunction) + "] = window[" + JSON.stringify(jsonpFunction) + "] || []).push([" + JSON.stringify(chunk.ids) + ",");
```

Comment:
The bootstrap makes a copy of this array, so any additions made after the bootstrap loads can't be seen.